app/helpers/firm_helper.rb

```diff
@@ -24,4 +24,12 @@ def minimum_pot_size_text(firm_result)
   def minimum_fixed_fee(firm_result)
     number_to_currency firm_result.minimum_fixed_fee, unit: '£', precision: 0
   end
+
+  def link_with_protocol(link)
+    if link =~ /^https?/
+      link
+    else
+      "http://#{link}"
+    end
+  end
```

Comment:
This helper is purely for firms websites. We have adjusted the validation rules to allow a firm to specify their web address with an optional protocol prefix.